theano/gpuarray/c_code/dimshuffle.c

```diff
@@ -0,0 +1 @@
+#section support_code
```

Comment:
GpuDimShuffle inherits from HideC and DimShuffle.  HideC will ensure that we don't try to reuse the C code from DimShuffle, so you could probably just add an empty `__init__` to GpuDimShuffle.

If you intend on rewriting GpuDimShuffle soon, even if not in this PR, I would say the current fix is fine.